packages/preview/src/browser/markdown/markdown-preview-handler.ts

```diff
@@ -229,9 +229,15 @@ export class MarkdownPreviewHandler implements PreviewHandler {
                         token.attrJoin('class', 'line');
                         token.attrSet('data-line', line.toString());
                     }
-                    return (originalRenderer)
-                        ? originalRenderer(tokens, index, options, env, self)
-                        : self.renderToken(tokens, index, options);
+                    // return (originalRenderer)
+                    //     ? originalRenderer(tokens, index, options, env, self)
+                    //     : self.renderToken(tokens, index, options);
+
+                    if (originalRenderer) {
+                        originalRenderer(tokens, index, options, env, self);
```

Comment:
I wonder if the changes in this file are the reason the CI fails?

Look at the checks (Travis and AppVeyor)

Reply:
I will look into them. Thanks!

Reply:
It turns out that *originalRenderer* returns a **string**, but somehow tslint thinks it returns **void**

Reply:
Just as an FYI, if you'd like to avoid using `// tslint:disable-next-line:no-void-expression`, you can refactor to the following which is logically equivalent:

```ts
if (originalRenderer) {
    originalRenderer(tokens, index, options, env, self);
} else {
    return self.renderToken(tokens, index, options);
}
```

Reply:
Nah, we left it as it was before, since `originalRenderer` doesn't really return `void`, but rather a `string`. It's weird.

Reply:
> Just as an FYI, if you'd like to avoid using `// tslint:disable-next-line:no-void-expression`, you can refactor to the following which is logically equivalent:
> 
> ```ts
> if (originalRenderer) {
>     originalRenderer(tokens, index, options, env, self);
> } else {
>     return self.renderToken(tokens, index, options);
> }
> ```

The problem is that *originalRenderer* actually returns a string (the rendered element) instead of void, so not returning it would cause issues.